### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by sam-watttime**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,104 @@
-# apiv3-sdk
-A software development kit for the WattTime API, showing basic examples of usage. 
+# About
+This SDK is meant to help users with basic queries to WattTime’s API (version 3), and to get data returned in specific formats (e.g., JSON, pandas, csv).
+
+Users must first [register for access to the WattTime API here](https://watttime.org/docs-dev/data-plans/).
```
Comment (nsteins) [inline README.md:4]: Actually, to register an account, they need to post to /register, right? Should we add that to the API?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,104 @@
...
+
+```
+
+Methods also exist to request historical forecasts, however these responses may be slower as the volume of data can be significant:
+```
+hist_forecasts = wt_forecast.get_historical_forecast_json(
+    start = '2022-12-01 00:00+00:00',
+    end = '2022-12-31 23:59+00:00',
+    region = 'CAISO_NORTH',
+    signal_type = 'healdh_damage'
```
Comment (geoffhancock) [inline README.md]: should be 'health_damage' (typo)

**Comment by nsteins** [comment]
I pushed some changes to the README. I think this is good, but lets add some documentation about registering once we add a register function. But no need to let this block this release.

The historical csv test is failing on gh actions, but I am able to run it successfully locally, so I suspect there's something wrong with the pathing somewhere? or maybe the tests don't have permission to write folders? Either way, we should fix the test before merging.

**Review by nsteins — APPROVED** [review]
Looks good for now. Two features we should add in the future (in separate PRs):
- a register function
- the ability to specify the output dir for csvs

**Pull request merged**